src/components/Header/Header.tsx

```diff
@@ -0,0 +1,32 @@
+import { FC } from 'react';
+
+type Props = {
```

Comment:
Consider renaming the `Props` type to `HeaderProps` to make it more descriptive and specific to the Header component. 🚀